src/stream/src/executor/lookup/impl_.rs

```diff
@@ -396,23 +364,31 @@ impl<S: StateStore> LookupExecutor<S> {
             let all_data_iter = match self.arrangement.use_current_epoch {
                 true => {
                     self.arrangement
-                        .state_table
-                        .iter_with_pk_prefix(&lookup_row)
+                        .storage_table
+                        .batch_iter_with_pk_bounds(
+                            HummockReadEpoch::NoWait(epoch_pair.curr),
```

Comment:
I have not known this before. IIUC, it is read from the global shared buffer in the compute node and it support the MVCC read(read with a specific epoch)?